run_no_timeout.py

```
@@ -0,0 +1,42 @@
+# Default to user_test.py if no script specified
+script = 'examples/user_test.py'
+args = sys.argv[1:]
+
+# If first arg looks like a script path, use it
+if args and args[0].endswith('.py'):
+    script = args[0]
+    args = args[1:]
+
+# Run without timeout
+try:
+    cmd = [sys.executable, script] + args
```

Ruff (0.11.9)

Consider `[sys.executable, script, *args]` instead of concatenation

Replace with `[sys.executable, script, *args]`

(RUF005)